### PR TITLE
feat: implementation of test-case #001: Validators submit large transactions

### LINKED ISSUE #58: testground/tests: implement TC-001
We already have all the presets done in `init_val.go` 
The only left to do is:

- Create more composition files reflecting data sets
- Adjust the bandwidths
- Better documentation tracing

Ref: 

1. https://github.com/celestiaorg/test-infra/blob/main/docs/test-plans/001-Big-Blocks/test-cases/tc-001-val-large-txs.md
2. #1 
3. #55

### PR DESCRIPTION
**Pull request by Bidon15**
- compositions are ordered by test-case -> data-sets
- adding a readme
- adapt manifest.toml

Should be reviewed after #57 is merged 
Close #58 #37

### CODE REVIEW

File: tests/tc-001-val-large-txs.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import (
+	appsync "github.com/celestiaorg/test-infra/tests/app-sync"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+// Test-Case #001 - Validators submit large txs
+// Description is in docs/test-plans/001-Big-Blocks/test-cases
+func ValSubmitLargeTxs(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
```
Comment (vgonkivs) [inline tests/tc-001-val-large-txs.go:11]: `func ValSubmitLargeTxs(runenv *runtime.RunEnv, initCtx *run.InitContext) (err error)`

So you can avoid creating a local variable at L12

Comment (Bidon15) [reply tests/tc-001-val-large-txs.go]: True. Thanks for the review. Done

**Review by vgonkivs — APPROVED** [review]
LGTM 🚀

**Review by SweeXordious — COMMENTED** [review]
LGTM, only some non blocking nits

**Review by SweeXordious — APPROVED** [review]
:rocket:

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Tests
+
+Each `*.go` follows the naming of the test-case in `docs/test-plan/test-plan-id/*`. 
+
+In order to not clutter the test-file with different participants' steps, each dedicated directory is created to 
+maintain steps of each types of participants. 
+
+The `common` directory is maintaining steps for a participant that plays the same role in every of the test-case(e.g. creating a validators set)
```
Comment (Wondertan) [inline tests/README.md:8]:
```suggestion
The `common` directory maintains steps for a participant that plays the same role in every of the test-case(e.g. creating a validators set)
```

**Review by Wondertan — APPROVED** [review]
Awesome!

**Pull request merged**